libs/core/src/lib/wizard/wizard.component.ts

```diff
@@ -308,16 +308,27 @@ export class WizardComponent implements AfterViewInit, OnDestroy {
                     const wizardNavigationHeight = this._elRef.nativeElement.querySelector(
                         '.' + WIZARD_NAVIGATION_CLASS
                     ).clientHeight;
-                    this.wrapperContainer.nativeElement.scrollTo({
-                        top: child.offsetTop - wizardNavigationHeight,
-                        behavior: 'smooth'
-                    });
+
+                    this.scrollTo(this.wrapperContainer.nativeElement, child.offsetTop - wizardNavigationHeight);
                 }
             });
             this._setUpScrollListener();
         }
     }
 
+    /** @hidden */
+    private scrollTo(containerElement: Element, distanceToScroll: number): void {
```

Comment:
Good job
one small comment
Can it be moved to the `utils` instead of having duplicates in two places?

Reply:
Agree with Yury, also should be prefixed with _ since it's private

Reply:
Done